components/inputgroup/metadata/combobox.yml

```diff
@@ -1,12 +1,45 @@
 name: Combobox
 SpectrumSiteSlug: https://spectrum.adobe.com/page/combo-box/
+sections:
+  - name: Migration Guide
+    description: |
+      ### New Textfield markup
+      InputGroup now uses the new Textfield markup. See the [Textfield migration guide](textfield.html#migrationguide) for more information.
+
+      ### Additional clases
+      The following classes must be added:
+
+      * `.spectrum-InputGroup-textfield` is now required on the Textfield outer element (`.spectrum-Textfield`)
+      * `.spectrum-InputGroup-input` is now required on the `<input>` element inside of Textfields (`.spectrum-Textfield-input`)
+      * `.spectrum-InputGroup-button` is now required on the FieldButton (`.spectrum-FieldButton`)
+
+      ### Renamed classes
+
+      * `.spectrum-Datepicker--rangeDash` renamed to `.spectrum-Datepicker-rangeDash`
+
+      ### Removed elements
+
+      * `.spectrum-Datepicker-focusRing` is no longer required and should be removed
```

Comment:
@LFDanLu this was removed completely as it's no longer required to render the styles correctly. Thoughts?

Reply:
Makes sense since v3 doesn't need it for the DateRangePicker from what I can see.